server/recceiver/cfstore.py

```diff
@@ -15,6 +15,7 @@
 import datetime
 import os
 import json
+from channelfinder import ChannelFinderClient
```

Comment:
This import was previously moved into the constructor so that the cfstore.py can be imported when channelfinder isn't present. 

6d990bac2bbd1d199ccbccd0e79c7abcf7a9e1b3